File: c++/control_flow/if_statement/if_else_multiple_statement.cpp
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */
// SPDX-License-Identifier: GPL-3.0-or-later

#include <iostream>

int main()
{
  if (1 > 2)
    std::cout << "1 > 2" << std::endl;
  else if (2 > 3)
    std::cout << "2 > 3" << std::endl;
  else
    std::cout << "!(1 > 2) && !(2 > 3)" << std::endl;

  if (1 > 2)
    std::cout << "1 > 2" << std::endl;
  else if (1 > 0)
    std::cout << "1 > 0" << std::endl;
  else
    std::cout << "!(1 > 2) && !(1 > 0)" << std::endl;

  if (2 > 1)
    std::cout << "2 > 1" << std::endl;
  else if (1 > 0)
    std::cout << "1 > 0" << std::endl;
  else
    std::cout << "!(2 > 1) && !(1 > 0)" << std::endl;
}

File: c#/control_flow/if_statement/IfElseMultipleStatement.cs
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

class IfElseMultipleStatement {
  public static void Main(string[] args) {
    if (1 > 2)
      System.Console.WriteLine("1 > 2");
    else if (2 > 3)
      System.Console.WriteLine("2 > 3");
    else
      System.Console.WriteLine("!(1 > 2) && !(2 > 3)");

    if (1 > 2)
      System.Console.WriteLine("1 > 2");
    else if (1 > 0)
      System.Console.WriteLine("1 > 0");
    else
      System.Console.WriteLine("!(1 > 2) && !(1 > 0)");

    if (2 > 1)
      System.Console.WriteLine("2 > 1");
    else if (1 > 0)
      System.Console.WriteLine("1 > 0");
    else
      System.Console.WriteLine("!(2 > 1) && !(1 > 0)");
  }
}
